### ISSUE TITLE
Add built-in vendor configuration for Parsely analytics

### ISSUE THREAD

**Issue by msukmanowsky** [ampproject/amphtml#1500]
## Objective

Add configuration tag for Parsely analytics. 

## Overview

PR to follow shortly including implementation.

## Proposed Vendor Configuration

```html
<amp-analytics type="parsely">
<script type="application/json">
{
  "vars": {
    "apikey": "example.com"
  }
}
</script>
</amp-analytics>
```

## Additional config variables requested

* screenAvailWidth (JavaScript property `screen.availWidth`)
* screenAvailHeight (`screen.availHeight`)
* colorDepth (`screen.colorDepth`)

### Client side sessionization

We don't see client-side sessionization currently supported in the AMP analytics spec and are inquiring as to whether inclusion would be possible. Parse.ly's client-side sessionization of pixel events sends a few additional parameters on all pixels:

* Session ID: an auto incrementing, 0 indexed, unsigned integer representing both a unique ID for the user's session and a count of total sessions.
* Session Initial URL: initial URL of the session
* Session Referrer: `document.referrer` property for the first page viewed in the session
* Session start time: Time in milliseconds since the Unix epoch for when the session began
* Last session start time: Time in milliseconds since the Unix epoch when the prior session began

**Comment by msukmanowsky** [comment]
@rudygalfi I can't assign labels to issues. Mind adding `INTENT_TO_IMPLEMENT` to this one?

**Comment by rudygalfi** [comment]
@msukmanowsky Added the label, thanks!

**Comment by rudygalfi** [comment]
Closed via #1595.
